lighthouse-core/lib/i18n/locales/ar-XB.json

```diff
@@ -1460,139 +1460,139 @@
   "lighthouse-core/report/html/renderer/util.js | warningHeader": {
     "message": "‏‮Warnings‬‏: "
   },
-  "stack-packs/packs/amp.js | efficient_animated_content": {
+  "node_modules/lighthouse-stack-packs/packs/amp.js | efficient_animated_content": {
```

Comment:
All of these keys have changed as the directory of the strings have changed. Is this breaking? Or shouldn't I care about this?

Reply:
I do prefer as you have it now, but @exterkamp will know if this will recreate work for our translators.

Reply:
No, these keys only matter to us :smile: Should be a no-op for translators.